CHANGELOG.md

```diff
@@ -5,7 +5,11 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Added
+- Add Twilio validation to make sure that post requests are coming from Twilio and relevant tests(CU-dgmfbv).
```

Comment:
Looks like you're missing a space before the brackets around the ClickUp number.

Reply:
Fixed & removed the bit about tests 😅